sampleapp/README-Container.md

```diff
@@ -52,6 +52,7 @@ spec:
   - port: 8080
     protocol: TCP
     name: rest
+    cohesityTag: ui
```

Comment:
Is this required param ? If yes, doesn't this break backward compatibility.

Reply:
Yes, it is a required parameter so that the 'Open App' button appears on the UI after the app installs.
By backward compatibility, are you referring to previous versions of cohesity software? I'm not sure if this parameter was a possible parameter in previous athena-spec files, but it is very much required now.

Reply:
I meant, the existing apps will now break because they don't have cohesityTag